packages/keyboard-shortcuts/src/store/selectors.js

```diff
@@ -13,6 +12,11 @@ import {
 	rawShortcut,
 } from '@wordpress/keycodes';
 
+/**
```

Comment:
I would feel way more comfortable if there were unit tests for those selectors, but this refactor is very straightforward so let's hope that CI is correct.